integration-tests/package-lock.json

```diff
@@ -697,12 +697,12 @@
         "node_modules/@frequency-chain/api-augment": {
             "version": "0.0.0",
             "resolved": "file:../js/api-augment/dist/frequency-chain-api-augment-0.0.0.tgz",
-            "integrity": "sha512-S0OzAtPqwx3L8BQwVhFfg5xheU2Dkv4n9ahQKhWVx9oW/EDkc9UERrBeTj0qISydGcUS56Jm1RtNLNJ67l2Gjg==",
+            "integrity": "sha512-Mr21sHhGlpOgwh65TEMco8oW6+t0ogk9+hDc5SuXaBZrVq4O6kQ1Pe6Hr1lcaixFu/AOc5tJTOdDHTYeCzd9RQ==",
```

Comment:
we probably do not need to check this in

Reply:
Yeah, the sha512 hash will change for every local build (I think).

Reply:
it was by accident, I forgot the tests would change it.